### PR TITLE
lubuild

### PR DESCRIPTION
**Pull request by tomlm**

### CODE REVIEW

File: packages/lubuild/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+```
+The same application name will be used in each azure region, with endpoints internal to it.
+
+## Generated .Dialog file for each lu file
+All language variations of a .LU files with the same prefix will get a .dialog file created
+which is a LUIS recognizer configured for it. 
+
+Example:
+```
+    models/tomlm/westus/Contoso.dialog1.lu.dialog <-- MultiLanguageRecognizer configured to use all of the languages
```
Comment (yochay) [inline packages/lubuild/README.md]: Why do we need app model per region? 

To be specific, do we need to generate a different model in different regions?

File: packages/lubuild/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+bind to the correct .dialog file for your model and runtime environment.
+
+## Environments
+When multiple people are working with models you want to be able to work with models
+independently from each other tied to the source control.  **LUBuild**
+
+By default, **LUBuild** uses the logged in user alias as the enviornment, so that you can
+simply create and work with local changes.  
+
+The environment is placed in *settings.luis.environment* configuration to control which
```
Comment (yochay) [inline packages/lubuild/README.md]: where is this file get saved / managed? Should we expect developers to edit this file?

File: packages/lubuild/README.md (hunk 1 above)
Comment (tomlm) [reply packages/lubuild/README.md]: it's per authoringRegions (there are 3, westus, westeurope and australiaeast)

File: packages/lubuild/README.md (hunk 2 above)
Comment (tomlm) [reply packages/lubuild/README.md]: it gets saved as a peer file to appsettings.json

**Pull request merged**